src/validation-engine.ts

```diff
@@ -41,6 +41,7 @@ export interface GenericValidator {
 }
 
 export type Validator = (NodeValidator | PropertyValidator | GenericValidator) & {
+  component: NamedNode
   prepareParam?(param: NamedNode, value: GraphPointer, allParams: Map<NamedNode, GraphPointer>): unknown | void
```

Comment:
`prepareParam` was not introduced in this PR, but I wonder about its purpose, I could not find any usage

Reply:
Good point. I cannot remember any more so it's probably good reason to remove if it's not needed now